install/debian_postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+PKG=dist-clang
+if [ "$1" = configure ] ; then
+        for f in clangd.conf
```

Comment:
To stick as close to code from debian wiki as possible. As I'm not a shell hacker :(

I could change to single variable if you wish, but this version seems more extensible to me.